### PR TITLE
Add docs for clickhouse-disks

### PR DESCRIPTION
**Pull request by myrrc**
### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/operations/utilities/clickhouse-disks.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+Program-wide options:
+
+* `--config-file, -C` -- path to ClickHouse config, defaults to `/etc/clickhouse-server/config.xml`.
+* `--save-logs` -- Log progress of invoked commands to `/var/log/clickhouse-server/clickhouse-disks.log`.
+* `--log-level` -- What [type](../server-configuration-parameters/settings#server_configuration_parameters-logger) of events to log, defaults to `none`.
+* `--disk` -- what disk to use for `mkdir, move, read, write, remove` commands. Defaults to `default`.
+
+## Commands
+
+* `copy [--diskFrom d1] [--diskTo d2] <FROM_PATH> <TO_PATH>`.
```
Comment (alexey-milovidov) [inline docs/en/operations/utilities/clickhouse-disks.md:20]: It's strange to have command-line arguments in camelCase. Maybe `--disk-to`?

Comment (myrrc) [reply docs/en/operations/utilities/clickhouse-disks.md:20]: Ok, let's change it

**Comment by alexey-milovidov** [comment]
Test has failed.

**Comment by myrrc** [comment]
My bad, didn't substitute all usages

**Pull request merged**